.husky/pre-commit

```diff
@@ -8,4 +8,4 @@ echo "\033[32mSpell Check passed\033[0m"
 echo
 echo "\033[33mStarting front matter precommit hook...\033[0m"
 ## Pre-commit hook that condenses section front-matter into master record.
-exec node .husky/front-matter-condenser.js update
```

Comment:
@jarrettgaither In general all files should end with a new line. The reason is a mix of convention but also some older software does not handle files not ending in a new line well. I am sure this functionally will not break anything but lets add the new line back.

There is a setting in VSCode to always add a new line ![vscode setting to add newline](https://i.sstatic.net/bWSmc.png)